transforms/ember-object/__testfixtures__/basic.output.js

```diff
@@ -0,0 +1,18 @@
+import EmberObject from "@ember/object";
+
+class MyObj extends EmberObject {
+  constructor() {
+    this.someProp = true;
+    this.biz = baz;
+  }
+
+  setFoo(bar = "") {
+    this.foo = bar;
+  }
+
+  setMyprop() {
+    this.set();
+  }
+}
+
+const myObj = new MyObj();
```

Comment:
It looks like this is not a 1-1 conversion of the above class, in the above we only create a class definition, not an instance. 

Also, we should definitely avoid converting to `new` syntax for the time being, at least until [RFC 337](https://github.com/emberjs/rfcs/pull/337) is decided on. If it is accepted, `new EmberObject()` will be deprecated, and `create` will be the only way to create an instance of an `EmberObject`.

Reply:
Thanks! I have updated both the codemod and fixtures